src/main/java/org/qubership/itool/context/FlowContextImpl.java

```diff
@@ -49,6 +47,7 @@
 
 import javax.annotation.Nullable;
 import javax.annotation.Resource;
+import com.google.inject.Inject;
```

Comment:
That's a wrong import.

For Guice 7.0 the import should better be `jakarta.inject.Inject` as it is a standard API rather than Google's-private `com.google.inject` one.

See https://github.com/google/guice/wiki/Guice700#jee-jakarta-transition